cpp_generation/modelSchemas/configs/HaConfig.json

```diff
@@ -290,6 +319,10 @@
       "num_type": "real_type",
       "dims": ["SS::pAG", "SS::NS"]
     },
+    "p_deaths_excess_nonaids": {
+      "num_type": "real_type",
+      "dims": ["SS::pAG", "SS::NS"]
+    },      
```

Comment:
Can children be exposed to non AIDS excess mortality? Or what is the reason for having both h_ and p_ outputs?